Justfile

```diff
@@ -314,65 +313,106 @@ publish-language-plugins-all: package-language-plugins-all
 compile-cfg-lib: publish-local-json publish-local-core
 	scala-cli --power compile besom-cfg/lib --suppress-experimental-feature-warning
 
+compile-cfg-containers: publish-local-cfg-lib publish-local-model
+	scala-cli --power compile besom-cfg/containers --suppress-experimental-feature-warning
+
 # Compiles besom-cfg k8s module
-compile-cfg-k8s: publish-local-cfg-lib
+compile-cfg-k8s: publish-local-cfg-lib publish-local-cfg-containers
 	just cli packages local kubernetes:4.17.1
```

Comment:
Is it intentional to have this version fixed here? What about publishing this module to maven? Are we going to always release besom-kubernetes for the latest k8s version and for 4.17.1? Otherwise this module will be unusable when downloaded from maven central, right?

Reply:
version is indeed fixed here and in project.scala for cfg-k8s, does it matter though? macro-stuff is going away, codegen is going to generate classes for the user without depending explicitly on besom-kubernetes.